deeppavlov/core/models/keras_model.py

```diff
@@ -170,4 +173,66 @@ def __call__(self, *x_batch, **kwargs) -> Union[List, np.ndarray]:
         """
         with self.graph.as_default():
             K.set_session(self.sess)
-            return self._net.predict_on_batch(x_batch, **kwargs)
\ No newline at end of file
+            return self._net.predict_on_batch(x_batch, **kwargs)
+
+
+class LRScheduledKerasModel(LRScheduledModel, KerasModel):
+    """
+    KerasModel enhanced with optimizer, learning rate and momentum
+    management and search.
+    """
+    def __init__(self, **kwargs):
+        if isinstance(kwargs.get("learning_rate_decay"), str):
+            LRScheduledModel.__init__(self, **kwargs)
```

Comment:
`AttributeError: 'KerasClassificationModel' object has no attribute 'load_path'` error is raised when running `python3 -m deeppavlov train intents_dstc2`

It happens because `KerasModel.__init__()` is not called and therefore attribute `self.load_path` is not set.

Reply:
fixed